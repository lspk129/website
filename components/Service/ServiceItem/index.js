import styled from 'styled-components';

const ServiceItem = styled.div`
  transition: all 0.5s ease-in-out;
  &:hover {
    background: ${props => props.theme.background};
    color: ${props => props.theme.text};
    .icons {
      color: ${props => props.theme.text};
    }
  };
  .icons {
    transition: all 0.5s ease-in-out;
    padding: 20px ;
    color: ${props => props.theme.background};
  }
  p {
    padding: 10px 50px;
  }
`;

export default ServiceItem;
