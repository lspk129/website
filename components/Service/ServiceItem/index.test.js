import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import ServiceItem from './';

describe('ServiceItem', () => {
  it('renders corretcly', () => {
    const wrapper = renderer.create(<ServiceItem />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    const wrapper = shallow(<ServiceItem />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveTagName('div');
  });
});
