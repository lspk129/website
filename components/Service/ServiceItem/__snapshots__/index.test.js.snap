// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceItem renders corretcly 1`] = `
<div
  className="sc-bdVaJa eIKJMt"
/>
`;
